README.md

```diff
@@ -1 +1,129 @@
-# avalanche-tooling-sdk-go
\ No newline at end of file
+# avalanche-tooling-sdk-go
+
+The official Avalanche Tooling Go SDK library.
+
+## Getting Started
+
+### Installing
+Use `go get` to retrieve the SDK to add it to your project's Go module dependencies.
+
+	go get github.com/ava-labs/avalanche-tooling-sdk-go
+
+To update the SDK use `go get -u` to retrieve the latest version of the SDK.
+
+	go get -u github.com/ava-labs/avalanche-tooling-sdk-go
+
+## Quick Examples
+
+### Subnet SDK Example
+
+This example shows how to create a Subnet Genesis, deploy the Subnet into Fuji Network and create
+a blockchain in the Subnet. 
+
+This examples also shows how to create a key pair to pay for transactions, how to create a Wallet
+object that will be used to create CreateSubnetTx and CreateChainTx and how to commit these 
+transactions on chain.
+
+```go
+package examples
+
+import (
+	"context"
+	"fmt"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/avalanche"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/keychain"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/subnet"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/teleporter"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/vm"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet"
+	"github.com/ava-labs/avalanchego/vms/secp256k1fx"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary"
+	"github.com/ava-labs/subnet-evm/core"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ethereum/go-ethereum/common"
+	"math/big"
+	"time"
+)
+
+// Creates a Subnet Genesis, deploys the Subnet into Fuji Network using CreateSubnetTx
+// and creates a blockchain in the Subnet using CreateChainTx
+func DeploySubnet() {
+	subnetParams := getDefaultSubnetEVMGenesis()
+	// Create new Subnet EVM genesis
+	newSubnet, _ := subnet.New(&subnetParams)
+
+	network := avalanche.FujiNetwork()
+
+	// Key that will be used for paying the transaction fees of CreateSubnetTx and CreateChainTx
+	// NewKeychain will generate a new key pair in the provided path if no .pk file currently
+	// exists in the provided path
+	keychain, _ := keychain.NewKeychain(network, "KEY_PATH")
+
+	// In this example, we are using the fee-paying key generated above also as control key
+	// and subnet auth key
+
+	// control keys are a list of keys that are permitted to make changes to a Subnet
+	// such as creating a blockchain in the Subnet and adding validators to the Subnet
+	controlKeys := keychain.Addresses().List()
+
+	// subnet auth keys are a subset of control keys
+	//
+	// they are the keys that will be used to sign transactions that modify a Subnet
+	// number of keys in subnetAuthKeys has to be more than or equal to threshold
+	// all keys in subnetAuthKeys have to sign the transaction before the transaction
+	// can be committed on chain
+	subnetAuthKeys := controlKeys
+	threshold := 1
+	newSubnet.SetDeployParams(controlKeys, subnetAuthKeys, uint32(threshold))
```

Comment:
I should set the control keys and threshold in one step, and the subnet auth key in a separate step

Reply:
maybe call it SetPermmisionedSubnetParams instead of set deploy params.
for the subnet auth step, just set it between subnet creation and blockchain creation, so it is clear where is
needed? and call it something that SetSubnetAuthorizationParams